app/controllers/validators_controller.rb

```diff
@@ -16,15 +16,19 @@ def index
     end
 
     watchlist_user = validators_params[:watchlist] ? current_user&.id : nil
+    session[:random_seed_val] ||= rand(0..1000)
 
     @validators = ValidatorQuery.new(watchlist_user: watchlist_user).call(
       network: validators_params[:network],
       sort_order: validators_params[:order],
       limit: @per,
       page: validators_params[:page],
-      query: validators_params[:q],
-      admin_warning: validators_params[:admin_warning],
-      jito: validators_params[:jito] == "true"
+      jito: validators_params[:jito] == "true",
+      params: {
```

Comment:
I'd rename it to query_params or something else, 'cause "params" can be misleading in controller. Also I would include jito in it, since it's also used in a base query